docs/api.rst

```diff
@@ -201,7 +224,7 @@ empty
 
   empty :: () -> Stream *
 
-Create a stream containing no events, which ends immediately.::
```

Comment:
I think removing the `::` affects the way the following lines are formatted, no?

Reply:
You’re correct, @davidchase. Two colons indicate that the following indented or quoted text is a literal block. I will add them back.

Reply:
One thing that's weird about the double-colon is that it ends up being rendered as a single colon.  So, we end up with perdiod-colon (like: "... which ends immediately.:"), which looks weird.  Not sure what to do about that ... remove the period in those cases?

Reply:
@briancavalier, they will _only_ be rendered as a colon when they’re not preceded by a space.